app/assets/stylesheets/modules/icons.scss

```diff
@@ -40,3 +40,16 @@
 .sul-i-remove-2 {
   color: $sul-unavailabe-icon-color;
 }
+
+.mediation-table {
+  .errored {
+    .request-status {
+      color: $sul-unavailabe-icon-color;
```

Comment:
Nice catch, we set that (miss-spelled) variable to a SUL Styles provided variable.  I'll fix that and push it up.